packages/ckeditor5-dev-ci/package.json

```diff
@@ -36,8 +36,11 @@
   },
   "dependencies": {
     "@octokit/rest": "^22.0.0",
+    "fs-extra": "^11.0.0",
```

Comment:
Instead of adding a dependency, can we just use `node:fs` and replace `readJson` with `readFile` and `JSON.parse()`?